apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1167,20 +1168,40 @@ export class PortfolioService {
     const userId = await this.getUserId(impersonationId, this.request.user.id);
     const userSettings = this.request.user.Settings.settings as UserSettings;
 
-    const { accounts, holdings, markets, summary } = await this.getDetails({
-      impersonationId,
-      userId,
-      withMarkets: true,
-      withSummary: true
-    });
+    const { accounts, holdings, markets, marketsAdvanced, summary } =
+      await this.getDetails({
+        impersonationId,
+        userId,
+        withMarkets: true,
+        withSummary: true
+      });
 
     const marketsTotalInBaseCurrency = getSum(
       Object.values(markets).map(({ valueInBaseCurrency }) => {
         return new Big(valueInBaseCurrency);
       })
     ).toNumber();
 
+    const marketsAdvancedTotalInBaseCurrency = getSum(
+      Object.values(marketsAdvanced).map(({ valueInBaseCurrency }) => {
+        return new Big(valueInBaseCurrency);
+      })
+    ).toNumber();
+
     const rules: PortfolioReportResponse['rules'] = {
+      RegionalMarketClusterRiskNorthAmerica:
```

Comment:
Please rename to `regionalMarketClusterRisk` and move below `economicMarketClusterRisk`